lib/reporters/lcov-reporter.js

```diff
@@ -0,0 +1,29 @@
+
+var Reporter = require('../reporter');
+
+var lcovRecord = function(data) {
+
+  var str = "";
+  str += 'SF:' + data.fileName + '\n';
```

Comment:
Thanks! This is my first foray into lcov, that's useful info.  Adding that link as a comment would be helpful for any future work.